### ISSUE TITLE
Gitter 3.1.0 Login Failing on Windows 7

### ISSUE THREAD

**Issue by jwilczek** [gitterHQ/gitter#1491]
When trying to login using Github account on Gitter, I receive the warning:

Hello! 
Gitter Windows Desktop App is requesting access to your account. 
Do you approve? 

When I click "ALLOW", the desktop app is minimized (to the icon tray).  When I open it from the icon tray, it shows the exact same warning.  I cannot get past this screen.

**Comment by MadLittleMods** [comment]
```
%LOCALAPPDATA%\Programs\Gitter\Gitter.exe --remote-debugging-port=1234
```

Visit [http://localhost:1234/](http://localhost:1234/) and choose "Gitter Apps"

![](http://i.imgur.com/aXRVnCz.png)

Do you see any errors in the developer console?

---

There is a specific desktop repo btw, https://github.com/gitterHQ/desktop

![](http://i.imgur.com/C99i1hk.jpg)

**Comment by jwilczek** [comment]
No errors,. just some mixed content warnings:

"Mixed Content: The page at 'https://gitter.im/login/oauth/authorize?redirect_uri=app%3A%2F%2Fgitter%2Foauth.html&response_type=code&client_id=windows-desktop-prod' was loaded over HTTPS, but requested an insecure resource 'http://beta-internal:8080/job/gitter-webapp-build-step/10117/artifact/output/maps/vendor.js.map'. This content should also be served over HTTPS."

Interesting it's trying to pull from beta-internal...  Ooops

**Comment by jwilczek** [comment]
Everytime I click "ALLOW" it launches a new instance of gitter.exe.

And, if I'm watching the console when I click "ALLOW", I see this:

![image](https://cloud.githubusercontent.com/assets/8474603/19863078/d1a3d112-9f60-11e6-80cd-fc518cd535e0.png)

**Comment by jwilczek** [comment]
I opened this on https://github.com/gitterHQ/desktop as well and haven't received a response from either side.  Is this is known issue?

**Comment by MadLittleMods** [comment]
@jwilczek  Are you behind a proxy? See https://github.com/gitterHQ/desktop/issues/49

**Comment by jwilczek** [comment]
No, not behind a proxy. But I'll try that solution nonetheless.

**Comment by jwilczek** [comment]
How do you enter the command? Simply pressing <enter> only seems to move to the next line. Is there something else needed to actually input the commands?
